_posts/2019-11-15-enforcing-database-transactions-with-rust.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: post
+title: Enforcing database transactions with Rust
+author: jgke
+excerpt: >
+  Rust's ownership model enables enforcing the use of database transactions,
+  while still allowing single queries without transactions.
+tags:
+ - Database
+ - Rust
+---
+
+Consider this Clojure snippet:
+
+```clojure
+(defn fetch-thing [user-id thing-id]
```

Comment:
nobody would write clojure like this... you need to pass the db in, not have it magically from the "environment"... I know this is a nitpicky thing